### PR TITLE
Added German translation

### PR DESCRIPTION
**Pull request by lanmarc77**
How about adding a German translation?

### CODE REVIEW

**Comment by johanstokking** [comment]
@lanmarc77 thanks! Our native speaker @kschiffer will review!

File: Mission.de.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#  The Things Network
+
+Das The Things Network ist ein offenes, dezentrales und "crowd sourced" Internet of Things (IoT) Datennetzwerk.
+
+## Vision
+
+Menschen, die damals ihre Netzwerke geöffnet und Datenaustausch über diese Netze zuließen, haben das Internet erschaffen. Dadurch entstand eine Synergie, die ein Wachstum ermöglichte, welches zu etwas führte, was heute mehr ist als die Summe seiner Einzelteile.
```
Comment (kschiffer) [inline Mission.de.md:7]:
```suggestion
Menschen, die damals ihre Netzwerke geöffnet und Datenaustausch über diese Netze zuließen, haben das Internet erschaffen. Dadurch entstand eine Synergie, die eine Entwicklung lostrat, die heute mehr ist, als die Summe seiner Teile.
```

File: Mission.de.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+Das The Things Network will zukünftig das gleiche mit dem IoT Netzwerk versuchen.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Im selben Geiste möchte _The Things Network_ ein weltweites, offenes IoT Netzwerk ermöglichen.
```

File: Mission.de.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Das Internet of Things benötigt hohe Kommunikationsreichweiten bei geringen Batteriekapazitäten und kostengünstige Lösungen zur Ermittlung von Sensordaten und der Kommunikation zwischen der Cloud und Einzelgeräten.
+
+Dem Ansatz von Moore‘s Gesetz folgend, verringern sich die Kosten von Computerkomponenten exponentiell. Die neue Generation von Netzwerkgeräten haben einen Preissegment erreicht, welches eine Finanzierung eines Netzwerkes durch die Masse (crowd) möglich macht.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Dem Ansatz von Moore‘s Gesetz folgend, verringern sich die Kosten von Computerkomponenten exponentiell. Die neue Generation von Netzwerkgeräten haben einen Preissegment erreicht, welches eine Finanzierung eines Community-basierten Netzwerkes durch ermöglicht.
```

File: Mission.de.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+Aktuelle Technologien machen es nun möglich ein kostengünstiges Datennetzwerk zu schaffen, welches durch eine freie Gemeinschaft der Nutzer bereitgestellt wird. Für ein Land wie beispielsweise die Niederlande, benötigt man weniger als 80 Gateways für eine flächendeckende Versorgung. Eine Stadt wie Amsterdam benötigt dafür nur 5 Gateways. Bei Kosten für ein Gateway, die mit denen eines durchschnittlichen Laptops vergleichbar sind, ist dies ein gut erreichbares Ziel.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Aktuelle Technologien machen es nun möglich, ein kostengünstiges Datennetzwerk zu schaffen, welches durch eine freie und offene Community bereitgestellt wird. Für ein Land wie beispielsweise die Niederlande, benötigt man weniger als 80 Gateways für eine flächendeckende Versorgung. Eine Stadt wie Amsterdam benötigt dafür nur 5 Gateways. Bei Kosten für ein Gateway, die mit denen eines durchschnittlichen Laptops vergleichbar sind, ist dies ein gut erreichbares Ziel.
```

File: Mission.de.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+Beispiele für Anwendungen reichen von kostengünstiger Positionserkennung in Schiffscontainern bis hin zu kleinen Halsbändern für Katzen, um diese immer ausfindig machen zu können.
+
+Jeder sollte in der Lage sein, Sensorendaten entweder zu versenden oder online abrufen zu können. Keine Monopole oder beschränkter Marktzugang sollte hier möglich sein.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Jeder sollte in der Lage sein, Sensorendaten entweder zu versenden oder online abrufen zu können. Diese Freiheit sollte durch keine privaten oder staatlichen Monopole beschränkt werden.
```

File: Mission.de.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+## Mission
+
+Unsere Mission ist der Aufbau eines dezentralen, Technologie unabhängigen IoT Netzwerkes, welches durch die Gemeinschaft der Nutzer betrieben wird und deren Eigentum ist.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Unsere Mission ist der Aufbau eines dezentralen, Technologieunabhängigen IoT Netzwerkes, welches durch die Gemeinschaft der Nutzer betrieben wird und deren Eigentum ist.
```

File: Mission.de.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+Wir erschaffen ein verteiltes Netzwerk, welches jegliche Formen der IoT Verbindungen unterstützt. Ein Netzwerk, welches von Beginn an demokratisch aufgebaut ist. Wir wollen sicherstellen, dass keine einzelne Macht IoT dominieren kann. Deshalb achten wir auf Netzwerkneutralität und eine Basis verteilte Installationen von Gateways, welche über ein Netzwerk zum Datenaustausch verbunden sind.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Wir erschaffen ein verteiltes Netzwerk, welches jegliche Formen der IoT Verbindungen unterstützt. Ein Netzwerk, welches von Beginn an demokratisch aufgebaut ist. Wir wollen sicherstellen, dass kein einzelner Akteur IoT dominieren kann. Deshalb achten wir auf Netzneutralität und stellen eine Basis von verteilten Installationen von Gateways bereit, welche über ein Netzwerk zum Datenaustausch verbunden sind.
```

File: Mission.de.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+Wir sind das Netzwerk.
+
+Das Fundament dieses Netzwerkes sind die folgenden Prinzipien:
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Das Fundament des _The Things Network_ sind die folgenden Prinzipien:
```

File: Mission.de.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+* **Ihre Daten sind Ihre Daten** - Daten sind Ende zu Ende verschlüsselt.
+* **Netzneutralität** - Jegliche Daten werden gleich behandelt.
+* **Quelloffen** - echnologie die der Gemeinschaft entspringt wird Quelloffen zur Verfügung gestellt.
+
+## Der Plan
+
+Wir erschaffen dieses Netzwerk selbst. Von den Netzwerkkomponenten, den Orten der Installation bis zur dezentralen Datenspeicherung.
+
+Das The Things Network wurde in Amsterdam gestartet. Wir haben bereits Partner gefunden, die ein Netzwerk errichten, welches die ganze Stadt abdeckt. Weitere Partner gründen auf dieser Basis Startups. Wir versuchen ein Weg aufzuzeigen, der als Blaupause für den Rest der Welt dienen kann.
```
Comment (kschiffer) [inline Mission.de.md:39]:
```suggestion
Das _The Things Network_ wurde in Amsterdam gestartet. Wir haben bereits Partner gefunden, die ein Netzwerk errichten, welches die ganze Stadt abdeckt. Weitere Partner gründen auf dieser Basis Startups. Wir versuchen ein Weg aufzuzeigen, der als Blaupause für den Rest der Welt dienen kann.
```

File: Mission.de.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+## Gemeinschaft
+
+Das Ziel unserer Gemeinschaft ist es, ein weltweit dezentrales Internet of Things Internet zu erschaffen.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Das Ziel unserer Community ist es, ein weltweites, dezentrales Internet of Things Internet zu erschaffen. 
```

File: README.de.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
+# Das „The Things Network“ Manifest
+
+Alles, was mit elektrischer Energie betrieben wird, wird auch irgendwann mit dem Internet verbunden werden.
+
+Diejenigen, die dieses Netzwerk kontrollieren, werden die Welt kontrollieren. *__Wir glauben, dass eine solche Macht nicht auf wenige Personen, Firmen oder Nationen verteilt sein sollte. Wir glauben eine solche Macht sollte über so viele Personen wie möglich verteilt werden.__* Um das zu erreichen haben wir das „The Things Network“ gegründet.
```
Comment (kschiffer) [inline README.de.md]:
```suggestion
Diejenigen, die dieses Netzwerk kontrollieren, werden die Welt kontrollieren. *__Wir glauben, dass eine solche Macht nicht auf wenige Personen, Firmen oder Nationen konzentriert sein sollte. Wir glauben, dass eine solche Macht an so viele Personen wie möglich verteilt werden muss.__* Um das zu erreichen, haben wir das „The Things Network“ gegründet.
```

File: README.de.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+
+Das „The Things Network“ ist eine quelloffen orientierte und unabhängige Bewegung mit den folgenden Grundsätzen:
+
+* Es verbindet Sensoren und Aktoren, welche als „Things“ bezeichnet werden, über „Things Gateways“  mit Computersystemen, die „Things Access“ genannt werden.
+* Der erste Teil einer solchen Verbindung findet drahtlos („Over The Air“) statt, der zweite Teil in einem Computernetzwerk („Over The Net“). Die dafür notwendigen Systeme, Konzepte und deren Aufbau nennen sich das „The Things Network“.
```
Comment (kschiffer) [inline README.de.md]:
```suggestion
* Der erste Teil einer solchen Verbindung findet drahtlos („Over The Air“) statt, der zweite Teil in einem Computernetzwerk („Over The Net“). Die dafür notwendigen Systeme, Konzepte und deren Aufbau nennen sich „The Things Network“.
```

File: README.de.md [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+* Jeder ist frei eigene „Things“ zu betreiben und diese mit einem „Things Gateway“  zu verbinden, unabhängig von dessen Eigentumsverhältnis.
```
Comment (kschiffer) [inline README.de.md:11]:
```suggestion
* Jeder ist frei, eigene „Things“ zu betreiben und diese mit einem „Things Gateway“  zu verbinden, unabhängig von dessen Eigentumsverhältnis.
```

File: README.de.md [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+* Jeder ist frei eigene „Things Gateways“ zu betreiben und diese mit einem „Things Access“ zu verbinden, unabhängig von dessen Eigentumsverhältnis. Diese „Things Gateways“ geben Zugriff zu allen „Things“ mit einem Ansatz der Netzneutralität alleinig limitiert durch die zur Verfügung stehende Bandbreite.
```
Comment (kschiffer) [inline README.de.md]:
```suggestion
* Jeder ist frei eigene „Things Gateways“ zu betreiben und diese mit einem „Things Access“ zu verbinden, unabhängig von dessen finanziellen Möglichkeiten. Diese „Things Gateways“ geben Zugriff zu allen „Things“ mit einem Ansatz der Netzneutralität, lediglich limitiert durch die zur Verfügung stehende Bandbreite.
```

File: README.de.md [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+* Jeder ist frei einen eigenen „Things Access“ zu betreiben und anonyme Verbindungen über das Internet bereitzustellen. Dieses „Things Access“ erlaubt Zugriff auf alle „Things Gateways“ mit einem Ansatz der Netzneutralität alleinig limitiert durch die zur Verfügung stehende Bandbreite. Dieses „Things Access“ erlaubt auch die Verbindung mit anderen „Things Access“, um Daten auszutauschen.
```
Comment (kschiffer) [inline README.de.md]:
```suggestion
* Jeder ist frei einen eigenen „Things Access“ zu betreiben und anonyme Verbindungen über das Internet bereitzustellen. Dieses „Things Access“ erlaubt Zugriff auf alle „Things Gateways“ mit einem Ansatz der Netzneutralität, lediglich limitiert durch die zur Verfügung stehende Bandbreite. Dieses „Things Access“ erlaubt auch die Verbindung mit anderen „Things Access“, um Daten auszutauschen.
```

File: README.de.md [hunk 16]
```diff
@@ -0,0 +1,18 @@
...
+* Die Verbindungsteile „Over The Air“ und „Over The Net“ sollen derart ausgestaltet sein, dass diese unabhängig von einem spezifischen Protokoll sind. Alle genutzten Protokolle sollen nicht proprietär, quelloffen und kostenfrei nutzbar sind.
+* Jeder, der ein „Things Access“ oder „Things Gateway“ betreibt, wird diese kostenfrei für alle damit verbundenen Geräte und Server bereitstellen.
+* Jeder der das „The Things Network“ benutzt, darf dies für beliebige Zwecke tun, beschränkt lediglich durch lokale Rechtsvorschriften, auf eigenes Risiko und mit dem Wissen, dass die Bereitstellung ohne Gewährleistung erfolgt und jederzeit ohne Ankündigung und ohne Angabe von Gründen beendet werden kann. Produkte und Services, die sich aus der Nutzung ableiten dürfen Sie frei für jeden anbieten, kommerzielle Zielen verfolgen, nur bestimmten Personen zur Verfügung stellen, ohne Gewinnerzielungsabsicht oder mit jeder anderen Absicht anbieten. Anbieter des „The Things Network“ erlegen ihren Nutzern keine Beschränkungen auf.
```
Comment (kschiffer) [inline README.de.md]:
```suggestion
* Jeder der das „The Things Network“ benutzt, darf dies für beliebige Zwecke tun, beschränkt lediglich durch die jeweils gültigen lokalen Rechtsvorschriften, auf eigenes Risiko und mit dem Wissen, dass die Bereitstellung ohne Gewährleistung erfolgt und jederzeit ohne Ankündigung und ohne Angabe von Gründen beendet werden kann. Produkte und Services, die sich aus der Nutzung ableiten, dürfen Sie frei für jeden anbieten, kommerzielle Zielen verfolgen, nur bestimmten Personen zur Verfügung stellen, ohne Gewinnerzielungsabsicht oder mit jeder anderen Absicht anbieten. Anbieter des „The Things Network“ erlegen ihren Nutzern keine Beschränkungen auf.
```

File: Mission.de.md [hunk 17]
```diff
@@ -0,0 +1,45 @@
+#  The Things Network
+
+Das The Things Network ist ein offenes, dezentrales und "crowd sourced" Internet of Things (IoT) Datennetzwerk.
```
Comment (kschiffer) [inline Mission.de.md]:
```suggestion
Das _The Things Network_ ist ein offenes, dezentrales und "crowd sourced" Internet of Things (IoT) Datennetzwerk.
```

**Review by kschiffer — CHANGES_REQUESTED** [review]
Hey @lanmarc77, thanks a lot for your contribution of a German translation! I have added some suggested changes, which I think will improve readability, fix a couple of typos and other issues. If you agree, you can integrate those and we can merge the PR.

**Comment by johanstokking** [comment]
Awesome, is this ready?

**Review by kschiffer — APPROVED** [review]

**Comment by johanstokking** [comment]
Thanks @lanmarc77

**Pull request merged**